tools/compare_versions/README.md

```diff
@@ -0,0 +1,7 @@
+Brick Version Comparison
+------------------------
+
+# How to use it?
+- `python tools/compare_versions/compare_versions.py --help` for getting help
+- `python --oldbrick 1.0.3 https://github.com/BrickSchema/Brick/releases/download/v1.0.3/Brick.ttl --newbrick 1.1.0 ./Brick.ttl
```

Comment:
Does this really work with URLs as well as local files? If so, that's really cool :)

Reply:
Yeah `rdflib` handles both in the same way. It's a good interface.